README.md

```diff
@@ -57,17 +57,26 @@ While the `web` frontend provides a performant, elegant, accessible and themeabl
 The backbone of this project is built by the following parts of the `packages`:
 - **client:** Generated TypeScript client for communications with the ownCloud Infinite Scale graph API
 - **container:** Static assets and rarely changing base files
+- **extension-sdk:** Provides utilities for developing and integrating custom extensions
 - **pkg:** Shared logic for various places inside the codebase
 - **runtime:** Central place of (user) authentication, provisioning of the user interface layout, client side storage, routing, theming, dependencies and (sub)application handling
 
 The repository's `packages` also contains the following apps, which can be en-/disabled via the `config.json`:
+- **activities:** An extension that provides a detailed activity stream, showing recent updates
+- **admin-settings:** An extension that allows administrators to manage users, groups, spaces and generic settings for their ownCloud instance efficiently
+- **app-store:** An extension that allows users to browse and download additional apps and extensions directly from the web interface
+- **epub-reader:** An extension for opening ebook files
 - **external:** An extension for creating, opening and editing files using the WOPI server
 - **files:** The default extension and core part of the project, responsible for file sync-and-share - up- and downloading, sharing with other users/groups or via links, version management and more
+- **importer:** Service to import your files from other cloud storage solutions
```

Comment:
You can omit this since the importer will move to the extensions repo anyway in this sprint 🤓

Reply:
Done!